samples/matter/lock/src/bolt_lock_manager.cpp

```diff
@@ -20,118 +20,52 @@ void BoltLockManager::Init(StateChangeCallback callback)
 	k_timer_init(&mActuatorTimer, &BoltLockManager::ActuatorTimerEventHandler, nullptr);
 	k_timer_user_data_set(&mActuatorTimer, this);
 
+	PinManager::Instance().Init();
+
 	/* Set the default state */
 	Nrf::GetBoard().GetLED(Nrf::DeviceLeds::LED2).Set(IsLocked());
 }
 
-bool BoltLockManager::GetUser(uint16_t userIndex, EmberAfPluginDoorLockUserInfo &user) const
+bool BoltLockManager::GetUser(uint16_t userIndex, EmberAfPluginDoorLockUserInfo &user)
```

Comment:
Regarding all these changes here, can we remove BoltLockManager methods and use Credentials Manager's ones directly? I can see that most of the methods just call almost the same API from Credential Manager.

Reply:
This is how I wanted to do it initially, bu then I though it would be more coherent if the `BoltLockManager` was the only direct user of the `CredentialsManager` and, as a result, only BoltLockManager was called in the zcl_callbacks.cpp. I will re-think it though, thanks.

Reply:
@ArekBalysNordic, @Damian-Nordic do you think my approach is ok?

Reply:
Both options are fine with me. I agree hiding credentials management may be clearer for existing users but on the other hand less code is always a good thing. Can't make up my mind really :)

Reply:
I'll leave as it is then :)